ibm_cloud_sdk_core/utils.py

```diff
@@ -117,6 +117,38 @@ def string_to_datetime(string: str) -> datetime.datetime:
     return val.replace(tzinfo=datetime.timezone.utc)
 
 
+def string_list_to_datetime_list(string_list: List[str]) -> List[datetime.datetime]:
```

Comment:
Could we shorten the name to be `string_to_datetime_list`?

Reply:
Sure, updated with the shortened names